### ISSUE TITLE
Prácticas 4 y 5

### ISSUE THREAD

**Issue by samtzai** [master-incar-ia/practicas-ia-2526-fernando-joseba#1]
Veo que algunas conclusiones que sacáis no se corresponden con lo que veo yo en las gráficas. Por ejemplo, vemos que el validation loss no baja a partir de 30, pero la gráfica solo llega hasta 30.  

¿Habéis entendido por qué se usan logits y no se usa la activación en la capa final?

Comentadme en el propio issue
